basster/legacy-bridge-bundle/1.0/config/packages/legacy_bridge.yaml

```diff
@@ -0,0 +1,7 @@
+basster_legacy_bridge:
+    # change path to the folder containing your legacy .php scripts.
+    legacy_path:    '%kernel.root_dir%'
+    # optional prepend script (see http://php.net/manual/en/ini.core.php#ini.auto-prepend-file)
+    # prepend_script: '/full/path/to/my/legacy/autoPrependFile.php' # can be ommited
```

Comment:
Im not a big fan of config in comments like this. Could you remove this and add a link to the configuration instead?

Reply:
otherwise move this information to your `Configuration` in an `->info()` node